packages/dso-toolkit/src/icons/spinner.scss

```diff
@@ -0,0 +1,3 @@
+// stylelint-disable-next-line block-no-empty -- allow empty block so that this icon is rendered in di.svg
+spinner {
```

Comment:
Wat vind ik ervan dat we lege scss in de code-base plaatsen ten behoeve van het genereren van 'iets'? Je mag toch verwachten dat we scss tbv styling hebben. Dat er styling in zit. Het commentaar is wel verhelderend. Zouden we niet een lijstje moeten hebben van icons die in di.svg terecht moeten komen? Want ik snap het nog steeds niet. Wat er in di.svg gaat heeft een scss. Behalve die in deze PR. Why? Het voelt krom. 
@tfrijsewijk What say you?

Reply:
Het is jouw voorstel, @tfrijsewijk, zie ik nu. 😶‍🌫️ ...

Reply:
Klopt.

Maar het is één van 1 voorstellen, zie comment https://github.com/dso-toolkit/dso-toolkit/issues/2844#issuecomment-2538825961

En eigenlijk stel jij een 4e oplossing voor. Nu ik dit zie ben ik ook niet per se heel enthousiast, maar ik weet het ook niet zo goed.

Het lastige van een lijstje is dat dit lijstje uit de pas kan gaan lopen met het filesystem: Er zijn twee bronnen die gecombineerd moeten worden.

Wat denken jullie van mijn suggesties?

Reply:
Ergens moet er een "lijstje" worden bijgehouden van de di.svg spritesheet. In de huidige opzet worden alle varianten al via een stylesheet gemaakt. Het lijkt mij dan het meest logisch om hierop verder te bouwen omdat je anders de scss varianten en het lijstje moet gaan combineren.

Persoonlijk vind ik een lege selecter met commentaar duidelijker dan een lege file (daar moet ook een stylinte-disable in voorkomen trouwens)

Reply:
Laten we geen lijstje gaan bijhouden, maar doen zoals het nu is opgezet met `.scss`-files met daarin een lege selector. 👍